src/core/objects/object_mover.cpp

```diff
@@ -124,19 +156,35 @@ void ObjectMover::move(qint32 dx, qint32 dy, bool move_opposite_handles)
 	}
 	
 	// Apply handle constraints
-	if (move_opposite_handles)
+	for (auto& constraint : handle_constraints)
 	{
-		for (auto& constraint : handle_constraints)
+		MapCoord anchor_point = constraint.object->getCoordinate(constraint.curve_anchor_index);
+		MapCoord moved_handle = constraint.object->getCoordinate(constraint.moved_handle_index);
+		MapCoordF to_hover_point = MapCoordF(moved_handle - anchor_point);
+		to_hover_point.normalize();
+		
+		MapCoord control = constraint.object->getCoordinate(constraint.opposite_handle_index);
+
+		// Check conditions for "corner point click-in" operation
+		if (move_opposite_handles == HandleOpMode::Click
+			&& constraint.anchor_is_corner)
```

Comment:
Please align `&&` with the `move_...` in the line above.

Reply:
Well, they are aligned with 4-space-sized tab. :)

However, looking again at this part I saw that the mode handling is somewhat confusing in its current form so I rewrote this part into a switch with fallthroughs, which is structuring the code in a more readable manner.